package.json

```diff
@@ -1337,7 +1337,7 @@
                 },
                 "jupyter.logging.level": {
                     "type": "string",
-                    "default": "error",
+                    "default": "debug",
```

Comment:
Are people going to get the info popup about debug logging being enabled?

Reply:
That code has been deleted